src/ui/qml/ContactList.qml

```diff
@@ -43,22 +43,55 @@ ScrollView {
             scroll.selectedContact = contactsModel.contact(contactListView.currentIndex)
         }
 
+        data: [
+            MouseArea {
+                anchors.fill: parent
+                z: -100
+                onClicked: contactListView.currentIndex = -1
+            }
+        ]
+
         section.property: "status"
-        section.delegate: Label {
-            height: implicitHeight + 8
-            width: parent.width
-            verticalAlignment: Qt.AlignVCenter
-            horizontalAlignment: Qt.AlignHCenter
-            text: {
-                switch (parseInt(section)) {
-                    case ContactUser.Online: return qsTr("Online")
-                    case ContactUser.Offline: return qsTr("Offline")
-                    case ContactUser.RequestPending: return qsTr("Requests")
-                    case ContactUser.RequestRejected: return qsTr("Rejected")
-                    case ContactUser.Outdated: return qsTr("Outdated")
+        section.delegate: Row {
+            width: parent.width - x
+            height: label.height + 4
+            x: 8
+            spacing: 6
+
+            Label {
+                id: label
+                y: 2
+
+                font.pointSize: styleHelper.pointSize
+                font.bold: true
+                font.capitalization: Font.SmallCaps
+                color: "#3f454a"
+
+                text: {
+                    // Translation strings are uppercase for legacy reasons, and because they
+                    // should correctly be capitalized. We go lowercase only because it looks
+                    // nicer when using SmallCaps, and that's a display detail.
```

Comment:
I did test that this works correctly for non-latin capitalizations, at least with all of the translations we have.